Utils/GamePath.cs

```diff
@@ -121,6 +121,20 @@ public static bool ExistsInDataAsPar(string path)
             return FileExistsInData(RemoveModPath(path) + ".par");
         }
 
+        public static bool IsXbox(string path)
+        {
+            try
+            {
+                using FileStream file = File.OpenRead(path);
```

Comment:
Shouldn't there be a better way of doing this? like checking the path itself, since UWP apps have a unique format for the folder name.